css/index.css

```diff
@@ -1,3 +1,374 @@
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 16px;
+   line-height: 18px;
+}
+
+.main-content .top-content {
+    display: flex;
+}
+
+.main-content .middle-img {
+    display: block;
+    margin: 30px auto;
+}
+
+.main-content .bottom-content {
+    display: flex;
+}
+
+.main-content .bottom-content p {
+    padding-right: 20px;
+}
+
+/* Contact Styles */
+
+.contact p {
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+/* Services Styles */
+
+.services-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    line-height: 1.3;
+}
+
+.services-content .services-header-img {
+    margin: 20px 0;
+}
+
+.services-content h1 {
+    font-size: 26px;
+    margin: 10px 0;
+}
+
+.services-content .services-text {
+ border-bottom: 1px solid black;
+ padding-bottom: 40px;
+ margin-bottom: 20px
+}
+
+.services-content .card {
+    padding: 20px;
+    border: 1px solid black;
+    width: 420px;
+    background: whitesmoke;
+    margin: 20px 0;
+}
+
+.services-content .card button {
+    padding: 10px 40px;
+    margin-top: 20px;
+    cursor: pointer;
+    background: white;
+    border-radius: 5px;
+    border: 1px solid black;
+}
+
+.services-content .card button:hover {
+    background: black;
+    color: white;
+} 
+
+.services-content .services-info ul {
+    margin: 20px;
+}
+
+.services-content .services-info ul li{
+    list-style: square;
+    padding: 5px;
+}
+
+.services-content .services-info {
+    display: flex;
+    flex-wrap: wrap;
+    border-top: 2px solid black; 
+    margin: 20px 0 35px 0;
+    padding-top: 40px; 
+}
+
+.services-content .services-info .facts {
+  width: 400px;
+  margin: 0 60px 20px 0;
+}
+
+.services-content .services-info .facts p {
+  margin-bottom: 20px;
+}
+
+.services-content .services-info .services-info-img {
+  width: 420px;
+  height: 300px;
+}
+
+/* For max-width at 800px */
+@media (max-width: 800px) {
+    .container {
+     width: 100%;
+    }
+
+    header {
+        display: flex;
+        flex-wrap: wrap;
+    }
+  
+    header nav {
+        width: 765px;
+        padding: 0 28px 0 28px;
+    }
+
+    header .logo {
+        display: block;
+        margin: 35px auto 0;
+        order: -1;
+    }
+
+    .cta {
+        display: flex;
+        justify-content: center;
+    }
+    
+    .cta img {
+        display: none;
+    }
+    
+    .cta .cta-text {
+        margin: 20px; 
+    }
+
+    .main-content {
+       margin: 70px 20px 35px 0;
+    }
+
+    .main-content .middle-img {
+       width: 100%;
+       padding: 0 0 0 20px;
+    }
+
+    .text-content {
+        margin: 20px;
+    }
+
+    .contact {
+        margin: 20px;
+    }
+}
+
+/*  For max-width at 500px */
+@media (max-width: 500px) {
+
+    header {
+        display: flex;
+        justify-content: center;
+    }
+
+    header nav {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
```

Comment:
If padding is set to 0 here, then it will get rid of the padding and and make it look a bit more like the design file for mobile view.